includes/ting_search_autocomplete.ajax.inc

```diff
@@ -0,0 +1,128 @@
+<?php
```

Comment:
Heh. It is a matter of consistency. E.g. `ting_search_autocomplete_settings` starts and ends with a blank line. Most other functions do not.

Most issues seem to have been cleaned up now.